cmd/virt-handler/Dockerfile

```diff
@@ -20,9 +20,18 @@ FROM fedora:26
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 
-RUN dnf -y install libvirt-client \
+RUN dnf -y install libvirt-client cyrus-sasl-scram \
```

Comment:
I don't know how far everything is, regarding to [scram-sha-256](https://tools.ietf.org/html/rfc7677), but right now, I think it is good enough, since it is the safest option we have and definitely way better than the unprotected socket before. @admiyo what do you say?